src/redux/selectors/index.js

```diff
@@ -112,24 +112,28 @@ export const allValidRecordsSortedByDateSelector = createSelector(
     .sort(sortByDate),
 );
 
-export const filteredRecordsSelector = createSelector(
+export const recordsWithShowOnlyFiltersSelector = createSelector(
```

Comment:
need to separate `showOnly` filters so this selector can be used with `resourceTypeFilters`